lib/Itho/CC1101.h

```diff
@@ -168,7 +168,7 @@ class CC1101
 		
 	//functions
 	public:
-		CC1101();
+		CC1101(int8_t CSpin = SS);
```

Comment:
Where is this awfully chosen define or global variable `SS` defined?
If it is in our code, let's rename it and if it is elsewhere, let's make a PR to change it.

Reply:
AFAICS, it is an Arduino defined constant, there are also the MISO, MOSI, etc. and D0, D1 constants. Not sure that will ever change 🤔

Reply:
Hmm OK, that will not likely change indeed. (the same as SuperSpeed USB3, which even worse as it often also has the charging symbol next to it)